### ISSUE TITLE
Update advanced search filter with new metadata labels 

### ISSUE THREAD

**Issue by cboelling** [MfN-Berlin/AnimalAudiogramDatabase#2]
The [advanced search page](https://animalaudiograms.museumfuernaturkunde.berlin/advanced) needs to be updated with the new labels for the metadata elements in [audiogram_metadata_scheme.md](../blob/main/resources/metadata/audiogram_metadata_scheme.md).

This might also give us an opportunity to check if the labels as they are used to define the scheme can be at once used for the database interface.

**Comment by alvarosaurus** [comment]
For starts, "Species" is not enough for the UI, since the UI has to list English and Latin names.

**Comment by cboelling** [comment]
> 
> 
> For starts, "Species" is not enough for the UI, since the UI has to list English and Latin names.
> Also, the UI should include information about the unit, e.g. age in months. As does the database.
> I'll see how far I get with the rest. However, if the scheme doesn't fit the use case, then we should change the scheme rather than the UI.

Agreed. Perhaps one way to proceed is to check if the term used in the metadata scheme can be used and appended by the additional information that's necessary, e.g.,

```species English vernacular name``` OR ```species (English vernacular name)```

The idea is that for those elements in th UI which correspond to elements in the metadata scheme the correspondence should be recognizable quickly and unambiguously.

We can certainly also adapt the terms in the metadata scheme if that turns out to be the better solution.

**Comment by alvarosaurus** [comment]
**Some comments on the names used in the database scheme**
This is the database scheme as it is implemented:
```
+--------------------------------+
| Tables_in_audiogrambase        |
+--------------------------------+
| audiogram_data_point           |
| audiogram_experiment           |
| audiogram_publication          |
| facility                       |
| individual_animal              |
| method                         |
| publication                    |
| sound_pressure_level_reference |
| taxon                          |
| test_animal                    |
+--------------------------------+
```
For future database implementations, please consider:
* database name "audiogrambase" is not bad, but there are datasets in the database which are not audiograms
* 'audiogram_data_point' could be just 'data_point'
* 'audiogram_experiment' could be 'experiment'
* 'audiogram_publication' is a linking table between 'audiogram_experiment' and 'publication', so should be 'experiment_publication'
* 'sound_pressure_level_reference' could be 'spl_reference'
* 'test_animal' is a linking table between 'individual_animal' and 'audiogram_experiment', so where does 'test' come from?

The general structure of the database is practical, but has limitations, for example when an audiogram is deleted, then undeleting it is not possible, as this would require a more normalized structure (as found in a CMS).

**Comment by cboelling** [comment]
>
> 
> For future database implementations, please consider:
> 
>     * database name "audiogrambase" is not bad, but there are datasets in the database which are not audiograms
> 
>     * 'audiogram_data_point' could be just 'data_point'
I agree.
> 
>     * 'audiogram_experiment' could be 'experiment'
I agree.
> 
>     * 'audiogram_publication' is a linking table between 'audiogram_experiment' and 'publication', so should be 'experiment_publication'
I agree.
> 
>     * 'sound_pressure_level_reference' could be 'spl_reference'
I would not recommend using abbreviations.
> 
>     * 'test_animal' is a linking table between 'individual_animal' and 'audiogram_experiment', so where does 'test' come from?
"test" is used here simply as a synonym for "experiment".
> 
> 
> The general structure of the database is practical, but has limitations, for example when an audiogram is deleted, then undeleting it is not possible, as this would require a more normalized structure (as found in a CMS).

I think, also in the light of our recent discussions to include data that reflect a variety of measurement objectives, i.e. critical ratios and others apart from auditory thresholds, we should conduct a review of our datamodel at some point.

The issue at hand for starters aims at harmonizing the terms used in the current version of the metadata scheme and the web interface.
